src/d3a/d3a_core/sim_results/device_statistics.py

```diff
@@ -152,7 +153,14 @@ def _gather_device_statistics(cls, area: Area, subdict: Dict, flat_result_dict:
 
         elif isinstance(area.strategy, FinitePowerPlant):
             market = list(area.parent.past_markets)[-1]
-            _create_or_append_dict(subdict, "production_kWh",
+            _create_or_append_dict(subdict, "energy_buffer_kWh",
```

Comment:
Can you please explain your decision for this name? Thanks!

Reply:
You are right, distinguishing between finite and infinite power sources and using `energy_buffer_kWh ` only for `InfiniteBus` and `CommercialProducer` strategy.